torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -1513,18 +1516,21 @@ ProcessGroupNCCL::~ProcessGroupNCCL() {
   }
 }
 
-bool ProcessGroupNCCL::dumpDebuggingInfo() {
+bool ProcessGroupNCCL::dumpDebuggingInfo(bool captureStackTrace/*=true*/) {
```

Comment:
nit: captureCppStackTrace?

Reply:
renamed to `includeStackTrace` to be more clear. Thanks for the suggestion!